docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,68 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Technical Documentation](https://github.com/grafana/technical-documentation) repository.
+
+   ```bash
+   git clone git@github.com:grafana/technical-documentation.git
+   ```
+
+1. Download [Vale](https://github.com/errata-ai/vale/releases).
+
+   {{% admonition type="note" %}}
+   Refer to the Linux installation steps at [GitHub Releases](https://vale.sh/docs/vale-cli/installation/#github-releases). Verify that you are downloading the latest build of Vale for Linux.
+   {{% /admonition %}}
+
+   {{< code >}}
+
+   ```linux
+   wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+   mkdir bin && tar -xvzf vale_2.28.0_Linux_64-bit.tar.gz -C bin
+   export PATH=./bin:"$PATH"
+   ```
+
+   ```macos
+   brew install vale
+   ```
+
+   {{< /code >}}
+
+1. Create a `vale.ini` file in your home directory or in a working directory with the following contents:
+
+   ```bash
+   StylesPath = /FULL_PATH_TO_REPO/technical-documentation/linters/vale
+   MinAlertLevel = suggestion
+
+   [*.md]
+   BasedOnStyles = Grafana
+   ```
+
+   Replace `FULL_PATH_TO_REPO` with the full path to the cloned Technical Documentation repository. For example, in Linux you could set StylesPath to `/home/username/git-repos/technical-documentation/linters/vale` and in macOS, you could set it to `/Users/username/git-repos/technical-documentation/linters/vale`. The path depends on where you cloned the git repository.
+
+1. Install the [Vale VS Code extension](https://marketplace.visualstudio.com/items?itemName=chrischinchilla.vale-vscode) in VS Code.
+
+   1. Start VS Code.
+   1. Press Ctrl+P, paste the following command, and press Enter.
+
+   ```
+   ext install ChrisChinchilla.vale-vscode
+   ```
+
+1. Configure the Vale VS Code extension.
+
+   1. Press Ctrl+Shift+X and select the Vale VS Code extension.
+   1. Select the gear icon.
+   1. Set `Vale › Vale CLI: Config` to the path to your `vale.ini` file. For example, on Linux that could be `/home/USERNAME/vale.ini` and on macOS, that could be `/Users/USERNAME/vale.ini`. The path depends on where you created the `vale.ini` file.
+   1. Set `Vale › Vale CLI: Path` to the path for the vale executable. For example, in Linux, that could be `/home/USERNAME/bin/vale` and on macOS, that could be `/usr/local/bin/vale`.
+
+1. Restart VS Code.
+
+Vale lints your current document every time you save your changes. The extension reports the linting results in two ways:
+
+1. In-line edit marks. You can hover your mouse cursor over the edit marks to view the vale warning or error.
```

Comment:
Is there a way to know it's a Vale error vs something else? I'm trying to test it using incorrect words from WordList.yml and I don't think it's working but I can't tell for sure.

Reply:
If it's working, you get a squiggly line under an error in the text plus something in the PROBLEMS report.
![image](https://github.com/grafana/writers-toolkit/assets/131809008/747ac599-1754-43a3-8e56-105f0f0fcbe0)